pkg/graveler/model.go

```diff
@@ -0,0 +1,15 @@
+package graveler
+
+import "github.com/treeverse/lakefs/pkg/kv"
+
+const (
+	tagsPrefix = "tags"
+)
+
+func TagPath(tagID string) string {
+	return kv.FormatPath(tagsPrefix, tagID)
```

Comment:
```suggestion
func TagPath(tagID TagID) string {
	return kv.FormatPath(tagsPrefix, tagID.String())
```

Reply:
done

Reply:
You should consider changing all the paths to []byte as we are going to change the keys type in the StoreMessage

Reply:
I'll do it after you will merge the changes, it can't compile now